platform/windows/os_windows.cpp

```diff
@@ -2621,6 +2616,7 @@ OS_Windows::OS_Windows(HINSTANCE _hInstance) {
 #ifdef RTAUDIO_ENABLED
 	AudioDriverManagerSW::add_driver(&driver_rtaudio);
 #endif
+	AudioDriverManagerSW::add_driver(&driver_dummy);
```

Comment:
Why does Windows get this added specifically when other platforms get it removed? It seems `AudioDriverManagerSW::initialize` already takes care of adding a dummy fallback.

Reply:
Done